notebooks/recommendation_systems/solutions/tfrs_basic_retrieval.ipynb

```diff
@@ -0,0 +1,1364 @@
+{
```

Comment:
<p>It is not very clear to me what this loss computes. Can you explain? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Added an explanation on <code>Retrieval</code>. Is it clearer?  </p>